### ISSUE TITLE
Error with count() function

### ISSUE THREAD

**Issue by outerbridge** [wpmetabox/mb-elementor-integrator#15]
Hi

Further to the fix for the Error with count() function https://github.com/wpmetabox/mb-elementor-integrator/pull/14/commits/c0cec1ab4d9fe4874072caee9c4f8a23ac995a37  I would propose changing line 257  in the GroupField.php file to check for a null value in $valueField prior to the other checks.  My proposal is:

```
        if ( ( null !== $valueField ) || ( is_array( $valueField ) && 0 == count( $valueField ) ) ) {
            return true;
        }
```

Without it, when I have a null value in $valueField, I'm seeing a fatal error as follows:
```
Fatal error: Uncaught Error: array_column(): Argument #1 ($array) must be of type array, null given
in /www/site_953/public/wp-content/plugins/mb-elementor-integrator/src/GroupField.php on line 270
```

Hope this helps.

**Comment by tuankiet2605** [comment]
Hi @outerbridge 
Can you send me this error checking video? I checked but no error.

Thanks.

**Comment by outerbridge** [comment]
Hi @tuankiet2605 

Thank you for your reply.

My apologies, I don't understand the question - which video are you referring to please?

When I used the standard GroupField.php file which has the following line 257 in it:
```
if ( is_array( $valueField ) && 0 == count( $valueField ) ) {
    return true;
}
```
I got the following error message:

```
PHP Fatal error:  Uncaught TypeError: array_column(): Argument #1 ($array) must be of type array, null given in /www/site_953/public/wp-content/plugins/mb-elementor-integrator/src/GroupField.php:270
```

But if I add a check for a null value to line 257:
```
if ( ( null !== $valueField ) || ( is_array( $valueField ) && 0 == count( $valueField ) ) ) {
    return true;
}
```

Then everything works as expected.

When I was trying to work out what was wrong, I saw that you'd already amended line 257 here -> https://github.com/wpmetabox/mb-elementor-integrator/commit/c0cec1ab4d9fe4874072caee9c4f8a23ac995a37 so I added my fix to that line.  The code appears to be excluding empty arrays, so I figured that null values for `$valueField` aren't wanted either.

Hopefully that helps explain the problem more clearly?  Please let me know what you need me to do.

Many thanks!

**Comment by tuankiet2605** [comment]
Hi @outerbridge 
Can you record video showing error when you setting elementor?

Thank you.

**Comment by outerbridge** [comment]
Hi @tuankiet2605 

The error appears on the site when I upgrade from version 2.0.8 of the plugin to version 2.1.1.  I've recorded a video to show you this -> https://youtu.be/H1Qx_zsUuEA

Many thanks!

**Comment by rilwis** [comment]
Fixed in #17
